subprojects/platform-jvm/src/main/kotlin/org/gradle/kotlin/dsl/ToolchainManagementExtensions.kt

```diff
@@ -23,9 +23,7 @@ import org.gradle.jvm.toolchain.JvmToolchainManagement
 
 /**
  * Provides statically defined accessors for configuring the "jvm" block in "toolchainManagement".
- * This block is added to <code>ToolchainManagement</code> dynamically, by the "jvm-toolchain-management"
- * plugin and normally the dynamic extension should generate accessors, but this doesn't work
- * at the level of Settings, hence the need for the static accessors.
+ * The "jvm-toolchain-management" plugin needs to be applied in order for these extensions to work.
```

Comment:
And if the plugin is not applied, are they visible?

Reply:
Yes. As far as I know, there's not much we can do about it atm.